src/PaymentGateways/PayPalCommerce/Banners/PayPalStandardToDonationsMigrationGlobalBanner.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Give\PaymentGateways\PayPalCommerce\Banners;
+
+use Give\PaymentGateways\Gateways\PayPalStandard\PayPalStandard;
+
+/**
+ * Class PayPalStandardToDonationsMigrationGlobalBanner
+ *
+ * Note: This class uses notice api to disable the banner for the user.
+ *
+ * @unreleased
+ */
+class PayPalStandardToDonationsMigrationGlobalBanner
+{
+    /**
+     * @unreleased
+     * @var string
+     */
+    private $bannerId = 'PayPalStandardToDonationsMigrationGlobalBanner';
+
+    /**
+     * @unreleased
+     * @return void
+     */
+    public function setHook()
+    {
+        // Bailout if user is not can not edit GiveWP settings.
+        if (! current_user_can('manage_give_settings')) {
+            return;
+        }
+
+        add_action('admin_enqueue_scripts', function () {
+            if (give_is_gateway_active(PayPalStandard::id())) {
+                wp_add_inline_script('give-admin-scripts', $this->getModalScript());
+            }
+        });
+    }
+
+    /**
+     * Render the banner.
+     *
+     * @unreleased
+     */
+    public function getModalScript(): string
+    {
+        if ($this->isBannerDisabledForUser()) {
+            return '';
+        }
+
+        $nonce = wp_create_nonce("give_edit_{$this->bannerId}_notice");
+
+        $modalTitle = esc_html__(
+            'PayPal Standard Deprecation',
+            'give'
+        );
+
+        $modalTitle = $this->getIcon() . "&nbsp;&nbsp;$modalTitle";
+
+        $modalSubHeading = esc_html__(
+            'Paypal Standard is no longer supported by PayPal',
+            'give'
+        );
+
+        $modalDescription = esc_html__(
+            'Migrate to PayPal Donations, which fully supports PayPal\'s latest API updates. As PayPal Standard is being deprecated, it will soon be removed from our platform.',
```

Comment:
I'd appreciate a content check here, @Benunc:
```suggestion
            'Migrate to PayPal Donations, which fully supports PayPal\'s latest API updates. As PayPal Standard is being deprecated, it will soon be removed from our platform. PayPal Standard will continue to work a while longer, but we strongly recommend migrating to PayPal Donations as soon as you can.',
```